### PR TITLE
Add Response.badRequest (400) constructor

### PR DESCRIPTION
**Pull request by point-source**

### CODE REVIEW

**Comment by kevmoo** [comment]
please add a test and update the changelog!

**Comment by kevmoo** [comment]
...and sign the CLA!

**Comment by point-source** [comment]
Done! Please review. Thanks @kevmoo

**Comment by kevmoo** [comment]
Also add a changelog entry....change `1.2.1-dev` to `1.3.0` – since this is adding a new feature. Also update pubspec.yaml

**Comment by point-source** [comment]
Need anything else done on this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,278 +1,281 @@
+## 1.3.0
+
+- Add Response.badRequest() constructor
```
Comment (kevmoo) [inline CHANGELOG.md:3]: I understand the desire, but please don't make unrelated changes!

**Review by kevmoo — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (point-source) [reply CHANGELOG.md:3]: Sorry, I don't understand what you are referring to. What are the unrelated changes I made? You asked me to add this to the changelog and change the version to 1.3.0 so I did. I'm confused.

Comment (point-source) [reply CHANGELOG.md:3]: Are you referring to the replacement of '*' with '-'? That was unintentional and was automatically done by my markdown formatter. I can certainly revert that if it makes a difference.

Comment (kevmoo) [reply CHANGELOG.md:3]: Yeah, it's best. 

Send a follow-up PR for those changes. Just shouldn't mix them.

Comment (point-source) [reply CHANGELOG.md]: Okay should be fixed now

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+## 1.3.0
+
+* Add Response.badRequest() constructor
 ## 1.2.1-dev
```
Comment (kevmoo) [inline CHANGELOG.md:4]: Delete this line. And roll the changes together. We'll just release 1.3.0

Comment (point-source) [reply CHANGELOG.md]: Okay, I _think_ I did what you were asking? Not 100% sure I followed that. You wanted me to delete the line that said 1.2.1-dev, yeah?

Comment (point-source) [reply CHANGELOG.md]: Ohh that's why you said to change it from 1.2.1-dev to 1.3.0. I understand now. Guess I'm just unfamiliar with that way of managing it but it makes sense

**Review by kevmoo — APPROVED** [review]

**Comment by kevmoo** [comment]
@point-source – you need to get the CLA bits going, first!

**Comment by point-source** [comment]
> @point-source – you need to get the CLA bits going, first!

I definitely signed that. Will check again but it was the first thing I did

**Comment by point-source** [comment]
Okay I think it was a username mismatch. Should be fixed now. Can you re-run the cla check?
